examples/upgrade.py

```diff
@@ -2,11 +2,16 @@
 """Asynchronous Python client for WLED."""
 
 import asyncio
+import sys
 
 from wled import WLED, WLEDReleases
 
 
 async def main() -> None:
+    if len(sys.argv) < 2:
+        print("Usage: python upgrade.py <ip_address>")
+        sys.exit(1)
+
```

Comment:
??

Reply:
Forgive my inexperience with python, if there is a better way please suggest rather than rejecting an improvement